src/app/components/search/SearchResultsTable/ItemThumbnail.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '@material-ui/core/Box';
+import VisibilityOffIcon from '../../../icons/visibility_off.svg';
+import styles from './ItemThumbnail.module.css';
+import MediaTypeDisplayIcon, { mediaTypeFromUrl } from '../../media/MediaTypeDisplayIcon';
+
+const ItemThumbnail = ({
+  type, picture, maskContent, url,
+}) => {
+  if (!maskContent) {
+    if (picture) {
+      return (
+        <Box display="flex" alignItems="center">
+          <img
+            className={styles.thumbnail}
+            alt=""
```

Comment:
@danielevalverde lets be good internet citizens and give the display an alt tag thats appropriate, maybe lets go with the item type? Unless you have another suggestion